decidim-assemblies/app/commands/decidim/assemblies/admin/create_assembly.rb

```diff
@@ -62,7 +62,27 @@ def assembly
             target: form.target,
             participatory_scope: form.participatory_scope,
             participatory_structure: form.participatory_structure,
-            meta_scope: form.meta_scope
+            meta_scope: form.meta_scope,
+            show_statistics: form.show_statistics,
+            purpose_of_action: form.purpose_of_action,
+            composition: form.composition,
+            assembly_type: form.assembly_type,
+            assembly_type_other: form.assembly_type_other,
+            creation_date: form.creation_date,
+            created_by: form.created_by,
+            created_by_other: form.created_by_other,
+            duration: form.duration,
+            included_at: form.included_at,
+            closing_date: form.closing_date,
+            closing_date_reason: form.closing_date_reason,
+            internal_organisation: form.internal_organisation,
+            is_transparent: form.is_transparent,
+            special_features: form.special_features,
+            twitter_handler: form.twitter_handler,
+            facebook_handler: form.facebook_handler,
+            instagram_handler: form.instagram_handler,
+            youtube_handler: form.youtube_handler,
+            github_handler: form.github_handler
```

Comment:
Oh my, so many fields

Reply:
Yes, I'm aware of the discussion. It was more like "Oh my, we are now sending a 40-elements Hash to create an Assembly" but hey, if it's what it takes... 🤷‍♂️